schemas/Metals/v0.0.1/schema.json

```diff
@@ -0,0 +1,1287 @@
+      "title": "BooleanResult",
+      "description": "A true/false or pass/fail result",
+      "type": "object",
+      "properties": {
+        "ResultType": {
+          "type": "string",
+          "const": "boolean"
+        },
+        "Value": {
+          "description": "The boolean value",
+          "type": "boolean"
+        },
+        "Description": {
+          "description": "Textual description of the boolean result",
+          "type": "string"
+        }
+      },
+      "required": ["ResultType", "Value"]
+    },
+
+    "StringResult": {
+      "title": "StringResult",
+      "description": "A textual result",
+      "type": "object",
+      "properties": {
+        "ResultType": {
+          "type": "string",
+          "const": "string"
+        },
+        "Value": {
+          "description": "The string value",
+          "type": "string"
+        },
+        "AllowedValues": {
+          "description": "List of allowed values (for enumerated results)",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      },
+      "required": ["ResultType", "Value"]
+    },
+
+    "RangeResult": {
+      "title": "RangeResult",
+      "description": "A result expressed as a range from minimum to maximum",
+      "type": "object",
+      "properties": {
+        "ResultType": {
+          "type": "string",
+          "const": "range"
+        },
+        "Minimum": {
+          "description": "The lower bound of the range",
+          "type": "number"
+        },
+        "Maximum": {
+          "description": "The upper bound of the range",
+          "type": "number"
+        },
+        "Inclusive": {
+          "description": "Whether the bounds are inclusive (true) or exclusive (false)",
+          "type": "boolean",
+          "default": true
+        }
+      },
+      "required": ["ResultType", "Minimum", "Maximum"]
+    },
+
+    "ArrayResult": {
+      "title": "ArrayResult",
+      "description": "A series of values typically representing a curve or multi-point measurement",
+      "type": "object",
+      "properties": {
+        "ResultType": {
+          "type": "string",
+          "const": "array"
+        },
+        "Values": {
+          "description": "Array of numeric values",
+          "type": "array",
+          "items": {
+            "type": "number"
+          },
+          "minItems": 1
+        },
+        "Parameters": {
+          "description": "Array of parameter values corresponding to the results",
+          "type": "array",
+          "items": {
+            "type": "number"
+          },
+          "minItems": 1
+        },
+        "ParameterName": {
+          "description": "Name of the parameter being varied",
+          "type": "string"
+        },
+        "ParameterUnit": {
+          "description": "Unit of the parameter",
+          "type": "string"
+        }
+      },
+      "required": ["ResultType", "Values"],
+      "oneOf": [
```

Comment:
I'm a little confused by this oneOf, could you explain the desired result? (It's possible my brain is just starved of caffeine at this moment 😅

Reply:
This has still to be tested in an update, which is required anyway to complete the rendering. Current version just covers the requirements of Everest.